JSTests/mozilla/ecma/Date/15.9.5.8.js

```diff
@@ -44,7 +44,8 @@
     var TZ_ADJUST = TZ_DIFF * msPerHour;
 
     // get the current time
-    var now = (new Date()).valueOf();
+    // var now = (new Date()).valueOf();
```

Comment:
Looking only at the code, it's unclear why this is commented out, so I think a link to the PR or Bugzilla should be added as a comment.